GGProject/workflow/models.py

```diff
@@ -1,3 +1,124 @@
 from django.db import models
 
-# Create your models here.
+# Registered employee
+class Employee(models.Model):
+
+    # First name of the employee
+    first_name = models.CharField(
+        null=True,
+        max_length=30
+    )
+
+    # Last name of the employee
+    last_name = models.CharField(
+        null=True,
+        max_length=30
+    )
+
+    # Employee's ID number that must be unique
+    emp_ID = models.IntegerField(
+        null=True,
+        unique=True,
+        verbose_name="Employee ID"
+    )
+
+    # Employee's email
+    emp_email = models.EmailField(
+        null=True,
+        max_length = 254,
```

Comment:
There should be consistency with spacing around the `=` throughout this file. All attributes should be in the form `max_length = 254` or `max_length=254` rather than a mix of both

Reply:
I looked and this seems to only be a problem for models.py - I added a commit to this pull request that fixes this inconsistency.